app/controllers/blueberry_cms/admin/menus_controller.rb

```diff
@@ -0,0 +1,50 @@
+module BlueberryCMS
+  module Admin
+    class MenusController < BlueberryCMS.page_admin_controller.constantize
+      helper BlueberryCMS::PagesHelper
+
+      def index
+        @menus = Menu.all
```

Comment:
We will most probably want to do some ordering or pagination or something, i don't think that `all` will be sufficient.